client/main.lua

```diff
@@ -137,98 +108,83 @@ local function CreateZone(type, number)
     local label
     local size
 
-    if type == "main" then
+    if type == 'main' then
```

Comment:
There seems to be a lot of repeating code following this elseif statements, you would be able to push more to the config and drastically reduce the size and complexity of these statements through:
```
if Config.Locations[type] then
<new code>
end
```

Reply:
Should I do these changes now or just wait until the resources all have ox_lib implemented then start refactoring

Reply:
I think this should be down to you, we can open up an issue so that we can keep track for a later date.

Reply:
I'd rather do it right from the beginning, I'll make another pr for it in like a hour